holidays/countries/venezuela.py

```diff
@@ -60,7 +60,7 @@ def _populate_public_holidays(self):
             # Declaration of Independence.
             self._add_holiday_apr_19(tr("Declaración de la Independencia"))
 
-        # https://bit.ly/3B4Xd1L
+        # https://web.archive.org/web/20220619063359/https://bit.ly/3b4xD1L
```

Comment:
I don't know why did we use bit.ly here, but here's the web archive-ed full source
```suggestion
        # https://web.archive.org/web/20250428122616/https://venezuelaenretrospectiva.wordpress.com/2016/05/01/1o-de-mayo-dia-del-trabajador-venezolano/
```